docs/get-started.md

```diff
@@ -237,28 +238,26 @@ installing Reader SDK for iOS, see [Reader SDK iOS Setup Guide].
    * `NSPhotoLibraryUsageDescription` : This app integrates with Square for card
       processing. Upload your account logo, feature photo and product images
       with the photos stored on your mobile device.
+
 1. Update the `application:didFinishLaunchingWithOptions:` method in your app
```

Comment:
Is this still the same method name? should it be `func application`?

Reply:
Done